# 工步信息
# 槽号
int32 pot_id
# 最小加工时间
int32 min_time
# 最大加工时间
int32 max_time